spec/services/proofing/resolution/progressive_proofer_spec.rb

```diff
@@ -198,101 +245,100 @@
       let(:applicant_pii) { Idp::Constants::MOCK_IDV_APPLICANT_SAME_ADDRESS_AS_ID }
 
       it 'returns a ResultAdjudicator' do
-        proofing_result = proof
-
-        expect(proofing_result).to be_an_instance_of(Proofing::Resolution::ResultAdjudicator)
-        expect(proofing_result.same_address_as_id).to eq(applicant_pii[:same_address_as_id])
+        expect(proof).to be_an_instance_of(Proofing::Resolution::ResultAdjudicator)
+        expect(proof.same_address_as_id).to eq(applicant_pii[:same_address_as_id])
       end
 
       context 'residential address and id address are the same' do
-        let(:aamva_proofer) { instance_double(Proofing::Aamva::Proofer) }
-        let(:residential_instant_verify_proof) do
+        let(:instant_verify_proofer_result) do
           instance_double(Proofing::Resolution::Result)
         end
+
         before do
           allow(instance).to receive(:with_state_id_address).and_return(transformed_pii)
           allow(instance).to receive(:state_id_proofer).and_return(aamva_proofer)
           allow(instance).to receive(:resolution_proofer).and_return(instant_verify_proofer)
-          allow(instant_verify_proofer).to receive(:proof).
-            and_return(residential_instant_verify_proof)
-          allow(residential_instant_verify_proof).to receive(:success?).and_return(true)
+          allow(instant_verify_proofer_result).to receive(:success?).and_return(true)
```

Comment:
I think it would be better to stub the property on the result in the constructor:
```ruby
        let(:instant_verify_proofer_result) do
          instance_double(Proofing::Resolution::Result, success?: true)
        end
```
or make `success` a `let` value